<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QDeviceSupplyStatus</class>
 <widget class="QWidget" name="QDeviceSupplyStatus">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>710</width>
    <height>278</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QDeviceSupplyStatus</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>-6</y>
     <width>710</width>
     <height>278</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="Supply">
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>691</width>
       <height>182</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,1">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <item>
       <widget class="QSupplyContainer" name="UnderDeck_Widget" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>180</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSupplyContainer" name="OutBoard_Widget" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>180</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QSupplyContainer" name="ISE"/>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>688</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="titles" stdset="0">
    <string>tit</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QSupplyContainer</class>
   <extends>QWidget</extends>
   <header>src/leonis/reagent/ch/QSupplyContainer.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
